include/lsst/jointcal/MeasuredStar.h

```diff
@@ -52,6 +52,12 @@ class MeasuredStar : public BaseStar
       { if (F)  F->MeasurementCount()++; fittedStar = F;
       }
 
+  void dump(std::ostream & stream = std::cout) const
+    {
+        BaseStar::dump(stream);
```

Comment:
I'll refrain from commenting on the uglyness of the C++ (i.e. why does `dump` exist in the first place).